src/ModelAdmin/CatalogPageAdmin.php

```diff
@@ -0,0 +1,129 @@
+            )->setHTMLID('Form_EditForm');
+        } else {
+            $form = parent::getEditForm();
+        }
+
+        $this->extend('updateEditForm', $form);
+        return $form;
+    }
+
+    /**
+     * @param null $id
+     * @param null $fields
+     * @param \SilverStripe\CMS\Model\SiteTree|\LittleGiant\CatalogManager\Extensions\CatalogPageExtension $model
+     * @return \SilverStripe\Forms\Form
+     */
+    protected function getCatalogEditForm($id = null, $fields = null, $model)
+    {
+        $originalStage = Versioned::get_stage();
+        Versioned::set_stage(Versioned::DRAFT);
+        $listField = GridField::create(
+            $this->sanitiseClassName($this->modelClass),
+            false,
+            $this->getList(),
+            $fieldConfig = GridFieldConfig_RecordEditor::create(static::config()->get('page_length'))
+                ->removeComponentsByType(GridFieldFilterHeader::class)
+                ->removeComponentsByType(GridFieldDeleteAction::class)
+                ->addComponent(new GridfieldPublishAction())
+        );
+        Versioned::set_stage($originalStage);
+
+        /** @var \SilverStripe\Forms\GridField\GridFieldDetailForm $detailForm */
+        $detailForm = $listField->getConfig()->getComponentByType(GridFieldDetailForm::class);
+        if ($detailForm !== null) {
+            $detailForm->setItemRequestClass(CatalogPageGridFieldItemRequest::class);
+        }
+
+        $sortField = $model->getSortFieldName();
+        if ($sortField !== null) {
+            $fieldConfig->addComponent(new GridFieldOrderableRows($sortField));
+        }
+
+        return Form::create(
+            $this,
+            'EditForm',
+            new FieldList($listField),
+            new FieldList()
+        )->setHTMLID('Form_EditForm');
+    }
+
+    /**
+     * Hook to update sort column on live versions of items after a sort has occured.
+     * @param $items
+     */
+    public function onAfterGridFieldRowSort($items)
+    {
+        /** @var \LittleGiant\CatalogManager\Extensions\CatalogPageExtension|\SilverStripe\CMS\Model\SiteTree $model */
+        $model = singleton($this->modelClass);
+        if (!$model::config()->get('automatic_live_sort')) {
+            return;
+        }
+
+        // if the sort is the default, then we should update SiteTree. If its a custom sort, update the model.
+        $sortField = $model->getSortFieldName();
+        $tableName = DataObjectSchema::singleton()->tableForField($this->modelClass, $sortField);
+        if ($tableName === null) {
+            throw new \Exception("Sort field {$sortField} could not be found in table hierarchy for {$this->modelClass}.");
+        }
+
+        foreach ($items as $item) {
+            if ($item instanceof $this->modelClass) {
+                DB::query("UPDATE `{$tableName}_Live` SET `{$model->getSortFieldname()}`=\"{$item->{$sortField}}\" WHERE ID={$item->ID}");
```

Comment:
ticks only work in mysql (will break other sql connectors). We prefer `"` instead (ansi sql92 standard).